src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160829_add_billing_project_status.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="what-git-says" id="add_billing_project_status">
```

Comment:
No. This information is "who wrote the liquibase changeset", not "who added the related scala code". Your comparison is invalid: they are different kinds of things. Liquibase changesets are by definition atomic; the variables (such as author) are also stored in the database, where there is no git annotate; and in this instance, there is no way (bar insanity or malice) for this particular duplication to become inconsistent.